Felienne/hedy
# Testing Hedy in Arabic Levels 1-5

Level 1-5 translations



**Description**

Adds the Arabic language to the list of all languages and to the languages menu. Includes the translation levels 1-5 in Arabic (Yaml files affected: level-defaults and keywords). 

**Fix for**

No Fix.

**How to test**

You should be able to see "العربية" as a language option in the drop down menu. When chosen the site should switch to RTL and the first five levels can be tested. 
Issue related: https://github.com/Felienne/hedy/pull/1703



**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [?] Only one thing is done in this pull request (specifically please try to refrain from mixing textual changes to the yamls from code changes)
